lisa/tools/reboot.py

```diff
@@ -166,16 +167,23 @@ def reboot(self, time_out: int = 600) -> None:
             try:
                 # check that vm has accessible ssh port
                 connected, _ = wait_tcp_port_ready(
-                    address=remote_node.public_address,
-                    port=remote_node.public_port,
+                    address=remote_node.connection_info[
+                        constants.ENVIRONMENTS_NODES_REMOTE_ADDRESS
+                    ],
+                    port=remote_node.connection_info[
+                        constants.ENVIRONMENTS_NODES_REMOTE_PORT
+                    ],
                     log=self._log,
                     timeout=20,
                 )
 
                 if not connected:
+                    node_ssh_port = remote_node.connection_info[
```

Comment:
using `remote_node.connection_info[constants.ENVIRONMENTS_NODES_REMOTE_PORT]` in LisaException below was creating very long line. That's I created this variable and used it in LisaException statement below.